lib/containers/k8s.pm

```diff
@@ -21,22 +21,18 @@ use registration qw(add_suseconnect_product get_addon_fullname);
 our @EXPORT = qw(install_k3s uninstall_k3s install_kubectl install_helm);
 
 sub install_k3s {
-    my $k3s_dowload_url = get_required_var("K3S_DOWNLOAD_URL");
-    assert_script_run("curl -LO $k3s_dowload_url");
-    assert_script_run("install -c -m 744 ./k3s /usr/local/bin/k3s");
+    assert_script_run("curl -sfL https://get.k3s.io | sh -");
     enter_cmd("k3s server 2>&1 | tee k3s.log &");
-    script_retry("grep 'Wrote kubeconfig /etc/rancher/k3s/k3s.yaml' k3s.log", delay => 20, retry => 10);
+    script_retry("cat /etc/rancher/k3s/k3s.yaml", delay => 20, retry => 10);
```

Comment:
does the `script_retry` still make sense here? it seems to me that `record_info`, `script_output` is better fit. wdyt?

Reply:
the idea is not to print the configuration, is to ensure that the configuration is created. This is replacing the grep because we cannot access to this log (or at least I didn't found) once this config yaml is created the cluster is accessible. It's a barrier to continue with the test

Reply:
I kept this without changes

Reply:
mmm I am going to check something... my words maybe are not accurate.. checking

Reply:
ok, I get it now, thanks. But anyway, the `assert_script_run` in line 24 will wait until the command finishes. Question is: when `curl -sfL https://get.k3s.io | sh -` is completed, is there some waiting time until the yaml is created? or is it created by that command at some point?

Reply:
The `curl...` line only download the binaries and scripts and install them. 
`k3s server` executes the kuberentes, this is the reason has to be executed in background

Reply:
> the idea is not to print the configuration, is to ensure that the configuration is created. This is replacing the grep because we cannot access to this log (or at least I didn't found) once this config yaml is created the cluster is accessible. It's a barrier to continue with the test

Not sure about the obstacle but it sounds like you could use `test -e` for just check the existence of the file. just an idea. i am not sure what the problem is exactly

Reply:
mmm yes, good point, thanks

Reply:
re-testing

Reply:
test passed